run/mc_features.py

```diff
@@ -137,117 +147,110 @@
+################################################################################
+# Ntuplizer steps
+
+if options.MVANtuplizer == True : # Use the Egamma-default MVANtuplizer code
 
    process.load('LowPtElectrons/LowPtElectrons/ElectronMVANtuplizer_cfi')
 
-else : # Use 'standalone' electron sequence and MVA ntuplizer code
+else : # Use custom Ntuplizer code
 
-   process.electronFeatures = cms.Sequence()
+   process.ntuplizer_seq = cms.Sequence()
+
+   # Track association by hits
    process.load('SimTracker/TrackAssociation/trackingParticleRecoTrackAsssociation_cfi')
-   # process.reconstruction *= process.simSiPixelDigis
-   process.electronFeatures *= process.tpClusterProducer
-   process.electronFeatures *= process.quickTrackAssociatorByHits
    process.quickTrackAssociatorByHits.useClusterTPAssociation = False
-   # process.quickTrackAssociatorByHits.associateStrip = False
-   process.electronFeatures *= process.trackingParticleRecoTrackAsssociation
+   process.ntuplizer_seq *= process.tpClusterProducer
+   process.ntuplizer_seq *= process.quickTrackAssociatorByHits
+   process.ntuplizer_seq *= process.trackingParticleRecoTrackAsssociation
    from SimGeneral.DataMixingModule.customiseForPremixingInput import customiseForPreMixingInput
    customiseForPreMixingInput(process)
 
-   # hack through the electron code
-   # make the tracker driven pass-though for pt > 0.5, produce preID
-   process.trackerDrivenElectronSeeds._TypedParameterizable__type = 'PassThroughTrackSeeds'
-   process.trackerDrivenElectronSeeds.MinPt = 1. ##@@
-   # process.trackerDrivenElectronSeeds.PtThresholdSavePreId = cms.untracked.double(0.) ##@@
-   process.trackerDrivenElectronSeeds.ProducePreId = True
-   process.trackerDrivenElectronSeeds.matchToGen = cms.bool(options.matchAtSeeding)
-   process.trackerDrivenElectronSeeds.genParticles = cms.InputTag("genParticles")
-   
-   # remove ECAL-driven seeds
-   process.ecalDrivenElectronSeeds._TypedParameterizable__type = 'EmptySeedProducer'
-   process.ecalDrivenElectronSeedsFromMultiCl._TypedParameterizable__type = 'EmptySeedProducer'
-   # set association to match against GSF, which is now run on every single track
-   process.trackingParticleRecoTrackAsssociation.label_tr = 'electronGsfTracks'
-
-   #
-   # ELECTRON ID
-   #
+   # Electron ID ...
    from PhysicsTools.SelectorUtils.tools.vid_id_tools import *
    dataFormat = DataFormat.AOD
    switchOnVIDElectronIdProducer(process, dataFormat)
    my_id_modules = [
       'RecoEgamma.ElectronIdentification.Identification.mvaElectronID_Fall17_noIso_V1_cff',
       'RecoEgamma.ElectronIdentification.Identification.mvaElectronID_Fall17_noIso_V2_cff',
    ]
-   
-   #add them to the VID producer
-   for idmod in my_id_modules:
-       setupAllVIDIdsInModule(process,idmod,setupVIDElectronSelection)
 
-   #change inputs
+   # Add them to the VID producer
+   for idmod in my_id_modules :
+      setupAllVIDIdsInModule(process,idmod,setupVIDElectronSelection)
```

Comment:
Are these tab to space changes? If so, does it still work with non-uniform indentation?
Personally I don't mind either option (slight preference for tabs, but I can live with anything), but I would suggest we choose and change only that in one commit to avoid huge diffs